src/utils/axiosInterceptors.ts

```diff
@@ -82,6 +94,14 @@ const setCacheResponse = async (response: any): Promise<any> => {
     }
 
     let cacheKey;
+
+    const expires = new Date();
+    expires.setSeconds(expires.getSeconds() + EXPIRES_IN_SECONDS);
+    response.headers = {
+      ...response.headers,
+      [EXPIRY_HEADER_KEY]: expires.getTime(),
+    };
```

Comment:
Another thing: I don't think this works? [Date.setSeconds](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setSeconds) expects "an integer between 0 and 59". But even if it does work, it might be best to use UNIX timestamps (`getTime()` and `setTime()`) directly to clearly communicate intent?

```
    const expiresMs = new Date().getTime() + EXPIRES_IN_SECONDS * 1000;
    response.headers = {
      ...response.headers,
      [EXPIRY_HEADER_KEY]: expiresMs,
    };
```

Reply:
Using seconds outside of the "expected" range does work as intended. Further in the docs it is also noted 
> If a parameter you specify is outside of the expected range, setSeconds() attempts to update the date information in the Date object accordingly.

Reply:
I have gone with your suggestion though, simply because of intent